### DISCUSSION TITLE
API for sharing workflows

### DISCUSSION THREAD

**Discussion by asemandeveloper** [OrchardCMS/OrchardCore#16219] [General]
Hi
Can I share orchard core workflows as an API with other software? How?

**Comment by Piedone** [comment]
If you mean whether another app can invoke a workflow of an OC app, then yes, You need to enable HTTP Workflows Activities for this and use the HTTP Request event.

**Reply by asemandeveloper** [reply]
I would like to write AI workflow module for Orchardcore which is like [DIFY](https://github.com/langgenius/dify) project. will you guide me

**Reply by Piedone** [reply]
You'll need to be more specific, then maybe I can help.

**Reply by asemandeveloper** [reply]
I want to use LLMs in Orchardcore project and build AI workflow on LLMs like DIFY project. Please see the [DIFY](https://cloud.dify.ai/signin) site.
I would be very grateful if you could help me and give me some useful tips.

**Reply by Piedone** [reply]
I understand this, but at this point, I can only write your project for you unless you ask specific questions :). I'd recommend you start learning about OC from the docs and https://docs.orchardcore.net/en/latest/resources/tutorials/.
